src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -1013,20 +1071,26 @@ static int server_receive_message(sd_event_source *s, int fd,
                 .msg_controllen = sizeof(control),
         };
         struct cmsghdr *cmsg;
-        ssize_t buflen, len;
+        ssize_t datagram_size, len;
         int r;
 
         assert(server);
 
-        buflen = next_datagram_size_fd(fd);
-        if (buflen < 0)
-                return buflen;
+        datagram_size = next_datagram_size_fd(fd);
+
```

Comment:
Please drop this blank line. The error check should be located right after the function call in general.